shub/copy_eggs.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+import os
+from six.moves.urllib.parse import urljoin
+import click
+import requests
+
+from shub.config import get_target
+from shub.fetch_eggs import fetch_eggs
+from shub.utils import decompress_egg_files, _deploy_dependency_egg
+
+SHORT_HELP = "Sync eggs from one project with other project"
+
+HELP = SHORT_HELP + """
+
+Fetch all eggs from one project and upload them to other project. This allows
+you to easily clone requirements from old project if you set up some new
+project in dash.
+"""
+
+
+@click.command(help=HELP, short_help=SHORT_HELP)
+@click.argument("target", required=False, default='default')
+@click.argument("new_project", required=True)
+@click.option("-m", "--copy-main", default=False, is_flag=True, help="copy main Scrapy project egg")
+def cli(target, new_project, copy_main):
+    project, endpoint, apikey = get_target(target)
+
+    copy_eggs(project, endpoint, apikey, int(new_project), copy_main)
```

Comment:
Users should be able to use named handles (as defined in `scrapinghub.yml`) for the target too, maybe even allowing different endpoints for source and target.
Something like `new_project, _, _ = get_target(new_project)` or `new_project, new_endpoint, new_apikey = get_target(new_project)`